src/robusta/cli/main.py

```diff
@@ -141,6 +122,9 @@ def gen_config(
                 )
             )
         )
+        if not verify_slack_channel(slack_api_key, cluster_name, slack_channel):
+            typer.echo(f"Invalid channel \"{slack_channel}\", Installation aborted.")
```

Comment:
I would try to elaborate a little more here.
i.e.: "We couldn't send our welcome message to channel {slack_channel}. Does this channel exist? Did you connect Robusta to the correct slack workspace?"